specific/account_asset_management_enhanced/models/account_asset_line.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+from openerp import fields, models, api
+
+
+class AccountAssetLine(models.Model):
+    _inherit = 'account.asset.line'
+
+    @api.multi
+    def write(self, vals):
+        write_ctx = self._context
+        if 'allow_asset_line_update' in write_ctx:  # prevent maximum recursive
+            return super(AccountAssetLine, self).write(vals)
+
+        for dl in self:
+            depreciation_line_ids = dl.asset_id.depreciation_line_ids
+            if len(depreciation_line_ids) == 1 and dl.type == 'create':
+                write_ctx = dict(self._context, allow_asset_line_update=True)
```

Comment:
For this, we have the same implementation in PABI2, can you search for keyword "MyModelLoopBreaker"
It is the keyword we use everywhere, and I try to have it consistent.